app/components/e-info-label.vue

```diff
@@ -93,6 +83,8 @@
 
 <style lang="scss">
   .e-info-label {
+    @include font($font-size: $font-size--14, $line-height: 14px, $font-weight: $font-weight--semi-bold);
```

Comment:
@geroldboehler FYI: If you use this mixin and define all the values in this order you don't have to use all the keys, can be simplified like this:
```
@include font($font-size--14, 14px, $font-weight--semi-bold);
```